### ISSUE TITLE
[XAML Controls Gallery(debug)->Date and Time->DatePicker/TimePicker] : Focus border is not clearly visible on few elements with tab navigation

### ISSUE THREAD

**Issue by v-amnat** [microsoft/Xaml-Controls-Gallery#677]
**Tags**
Accessibility, A11yCT, A11yMAS, A11yLowImpact, MAS2.4.7, KeyboardNavigation,
 A11ySev3, XAMLControlsGallery, WinUI, Win32, Wipro

**Test Environment**
1. XAML Controls Gallery(debug)  version - 2.0.21.0
2. Win 32/ OS build- 21343.rs_Prerelease.210320-1757

**Steps to reproduce the bug**

1. Launch the XAML Controls Gallery (Debug) App
2. Go to the Date and Time-> DatePicker.
3. Hit on Tab key to traverse the "DatePicker(Carousals)".
4. Observe the Kb Focus on "Pick a Date" element with tab navigation.
5. Go to the Date and Time-> TimePicker. 
6. Hit on Tab key to traverse the "TimePicker(Carousals)".
7. Observe the Kb Focus with tab navigation on below elements.
     > -  Time Picker
     > -  Arrival Time
     > -  24 Hour Clock

**Actual behavior**
With tab navigation focus is not visible on above mentioned elements. On Mouse hover elements are focused prominently and border is also seen.

**Expected**
Focus indicator(s) must be clearly visible on each interactive element when navigating the page using the tab key.

**User Impact** 
Keyboard only users need to know which element has focus so they can track their navigation through and interaction with interactive controls. Note that non-interactive content will not get keyboard focus and cannot be navigated to with the keyboard alone.

**MAS Reference**
MAS 2.4.7 – Focus Visible

**Attachments**
[Focus and border are not clearly visible.zip](https://github.com/microsoft/Xaml-Controls-Gallery/files/6310037/Focus.and.border.are.not.clearly.visible.zip)

**Comment by chigy** [comment]
This is already fixed in WinUI
